api.md

```diff
@@ -8,10 +8,9 @@ The Supervisor is the main object used to coordinate messaging and error handlin
 
 ### Attributes
 
-- `project_name` (str): The name of the client project using Supervisor; used to report version in global exception handler messages
 - `message_handlers` (List): Notifications will be sent via all handlers in this list
-- `logger` (Logger): Logger object to use in global exception handler
-- `log_path` (Path): Path to a logfile to send in exception messages
+- `logger` (Logger, optional): Logger object to use in global exception handler
+- `log_path` (Path, optional): Path to a logfile to attach to exception messages
```

Comment:
**question**: exceptions messages are the emails or whatever thing the handler decides to do?

Reply:
here, message means email, yeah. I'll clarify this.